pages/projects.html

```diff
@@ -30,11 +30,26 @@
       </div>
     </nav>
 
-    <div>
-      <p>Cookie Clicker Reimagining</p>
-      <img src="../images/CookieClickScreenshot.png" width="400px">
-      <p> To Do List </p>
-      <img src="../images/HugeManateeScreenshot.png" width="400px">
+    <div class="projects-container">
+      <h1>Cookie Clicker Reimagining</h1>
+      <a href="https://github.com/lilvina/cookie-game">Github</a></li>
+      <p>
+        This was one of the earlier projects that I've done at Learners Guild.
+        It was my first experience with local storage.
+      </p>
+      <div>
+        <img src="../images/CookieClickScreenshot.png" width="400px">
+      </div>
+      <h1> To Do List </h1>
+      <a href="https://github.com/breyana/huge-manatee-to-do-list">Github</a></li>
+      <p>
+        This was my first experience using React.
+        The fun part was allowing the user to click the task's text to edit the task
+        and have it saved on the backend.
```

Comment:
Trying to not run the lines too long past the 50 character mark so it's possibly easier to read